src/main/java/gnosis/ui/GnosisUI.java

```diff
@@ -0,0 +1,180 @@
+package gnosis.ui;
+
+import gnosis.controller.GnosisController;
+import gnosis.model.Task;
+import gnosis.util.GnosisConstants;
+import gnosis.util.GnosisException;
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * The GnosisUI handles the user interaction with user.
+ * It focuses on listening for input and displaying output.
+ *
+ * @author Pawandeep Singh
+ * */
+public class GnosisUI extends AnchorPane {
```

Comment:
Class names should be in camelCase, so perhaps this could be changed to `GnosisUi`